File: Day2 Operators/Operators.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */ 
    double mealCost, tip, tax, totalCost;
    int tipPercent, taxPercent, round;
    cin >> mealCost;
    cin >> tipPercent;
    cin >> taxPercent;
    tip = mealCost * tipPercent/100;
    tax = mealCost * taxPercent/100;
    totalCost = mealCost + tip + tax;
    cout << "The total meal cost is " <<nearbyint(totalCost) <<" dollars."<< endl;
    return 0;
}

File: Day2 Operators/Operators.java
import java.util.*;
import java.math.*;

public class Arithmetic {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double mealCost = scan.nextDouble(); // original meal price
        int tipPercent = scan.nextInt(); // tip percentage
        int taxPercent = scan.nextInt(); // tax percentage
        scan.close();
        double tip, tax;
        // Write your calculation code here.
      tip = mealCost * tipPercent/100;
    tax = mealCost * taxPercent/100;
    //totalCost = mealCost + tip + tax;
        // cast the result of the rounding operation to an int and save it as totalCost 
        int totalCost = (int) Math.round(mealCost + tip + tax);
      
        // Print your result
        System.out.println("The total meal cost is " + totalCost + " dollars.");
    }
}